src/libtiled/hexagonalrenderer.cpp

```diff
@@ -67,23 +67,52 @@ QRect HexagonalRenderer::mapBoundingRect() const
 {
     const RenderParams p(map());
 
+    QRect mapBounds;
+
+    LayerIterator iterator(map());
+    while (Layer *layer = iterator.next()) {
+        if (TileLayer *tileLayer = dynamic_cast<TileLayer*>(layer))
+            mapBounds = mapBounds.united(tileLayer->bounds());
+    }
+
+    if (mapBounds.size() == QSize(0, 0))
+        mapBounds.setSize(QSize(1, 1));
```

Comment:
We discussed this before. If the size is set to (0, 0) then after creating a new map, the 'weird' scrolling problem starts.